app/src/main/java/co/condorlabs/customcomponents/simplecamerax/CameraActivity.kt

```diff
@@ -31,6 +45,7 @@ class CameraActivity : AppCompatActivity(), SimpleCameraXFragment.OnCameraXListe
 
     private fun setupScreenProperties() {
         with(cameraConfig) {
+            if (fitToScreen) setFitToScreen()
```

Comment:
The code agreement says don't do that. https://github.com/cebroker/guidelines/blob/master/mobile/AndroidCodeGuideLines.md